submissions/ZjeniiaSt/html-css-popup/index.html

```diff
@@ -0,0 +1,146 @@
+                    ><span>Translate</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/videos.png" alt="" /> </a
+                    ><span>Video</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/search.png" alt="" /></a
+                    ><span>Search</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/photos.png" alt="" /></a
+                    ><span>Photo</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/apps.png" alt="" /></a
+                    ><span>Apps</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/documents.png" alt="" /></a>
+                    <span>Documents</span>
+                  </li>
+                </ul>
+
+                <input type="checkbox" id="submenu" class="menu-button" />
+                <label for="submenu" class="buttonfirst">
+                  <span class="button">More</span>
+                </label>
+
+                <ul class="menu-list submenu-list">
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/chat.png" alt="" /></a
+                    ><span>Chat</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/classroom.png" alt="" /></a
+                    ><span>classroom</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/collection.png" alt="" /></a
+                    ><span>Collection</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/mars.png" alt="" /></a
+                    ><span>Mars</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/blogs.png" alt="" /></a
+                    ><span>Blogs</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/promo.png" alt="" /></a
+                    ><span>Promos</span>
+                  </li>
+                </ul>
+
+                <div class="menu-button buttonsecond">
+                  <a href="#">Even more</a>
+                </div>
+              </div>
+            </li>
+
+            <li class="nav-item">
+              <a href="#">
+                <img src="./images/bell.png" alt="" />
+              </a>
+            </li>
+
+            <li class="nav-item">
+              <a href="#">
+                <img src="./images/194938.png" />
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+  </body>
+</html>
```

Comment:
Add empty lines at the end of each file. I believe you can set your VSCode to do so for you each time you save a file.